components/omnibar/README.md

```diff
@@ -1,5 +1,2 @@
 # Omnibar
-The Omnibar is St. Jude Cloud's primary navigation. It is used on every app or site, always at the top of each view.
-
-### Sketch File
-To use the omnibar in your sketch files, you can go to the [design.stjude.cloud sketch.cloud](https://www.sketch.com/team/NAv4/p/KYEe) directory and choose which screen size template you would like to start your prototype or design.
\ No newline at end of file
+The Omnibar is St. Jude Cloud's primary navigation. It is used on every app or site, always at the top of each view.
```

Comment:
Not needed anymore, as the options to add the Omnibar should be in the Figma file.